_docs/additional-features.md

```diff
@@ -274,9 +274,9 @@ POST /v1/transactions
 
 {% include language-tabbar.html prefix="auto-cancellation-trait" raw=data-raw %}
 
-We can also enable auto refund by default across all transactions created by you. If this is of interest, please contact our team so we can configure your account as such. If the feature is enabled, then it can be disabled on a per-transaction basis by specifying `"auto_refund": false` in the `traits` section.
+Auto refund is enabled by default across all transactions created by you. The feature can be disabled on a per-transaction basis by specifying `"auto_refund": false` in the `traits` section.
 
-Once the trait is enabled and 24 hours have elapsed since the transaction has been funded without a successful payout, we will cancel the transaction. If the transaction was paid from the account balance, the funds will also be immediately returned to the account balance and can be used immediately to fund new transactions.
+If the trait is enabled and 24 hours have elapsed since the transaction has been funded without a successful payout, we will cancel the transaction. If the transaction was paid from the account balance, the funds will also be immediately returned to the account balance and can be used immediately to fund new transactions.
```

Comment:
```suggestion
However if the trait is enabled and 24 hours have elapsed since the transaction has been funded without a successful payout, we will cancel the transaction. If the transaction was paid from the account balance, the funds will also be immediately returned to the account balance and can be used immediately to fund new transactions.
```